docs/ref-info/ref-operators-public-private.md

```diff
@@ -17,7 +17,13 @@ All instances of the Operator Service are designed with rigorous protections in
 
 There can be multiple instances of the Operator Service, operated by multiple participants. Each of these participants is known as a UID2 Operator.
 
-A UID2 Operator is simply an organization that runs the Operator Service. Operators receive and store encryption keys and salts from the UID2 Core Service, salt and hash personal data to return raw UID2s, encrypt raw UID2s to generate UID2 tokens, and distribute UID2 token decryption keys.
+A UID2 Operator is simply an organization that runs the Operator Service. Operators periodically perform multiple tasks including the following:
+
+- Receive and store up-to-date encryption keys and salts from the UID2 Core Service.
+- Salt and hash personal data to return raw UID2s.
+- Encrypt raw UID2s to generate UID2 tokens.
+- Distribute UID2 token decryption keys.
+- Download the latest user opt-out information from the UID2 <a href="glossary-uid#gl-opt-out-service">Opt-Out Service</a>.
```

Comment:
should we mention operator provides a list of endpoints and link to https://unifiedid.com/docs/endpoints/summary-endpoints

Reply:
Good idea -- done -- thx.